code/src/index.css

```diff
@@ -1,13 +1,146 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #181818;
+}
+
+.app {
+  display: flex;
+
+}
+
+aside {
+  width: 250px;
+  height: 100vh;
+  background-color: #111111;
+  position: sticky;
+  top: 0;
+  padding: 20px;
+  overflow: auto;
+}
+
+aside ul {
+  list-style: none;
+}
+
+aside li {
+  padding: 10px 0;
+}
+
+aside p {
+  color: #a0a0a0;
+}
+
+.hidden {
+  display: none;
+}
+
+aside p:hover {
+  color: #fff;
+}
+
+aside p:hover .hidden {
+  display: inline;
+}
+
+main {
+  flex-grow: 1;
+}
+
+h2 {
+  font-family: "Helvetica", "Arial", sans-serif;
+  font-size: 20px;
+  color: #fff;
+  margin: 20px 0;
+}
+
+h3 {
+  font-family: "Helvetica", "Arial", sans-serif;
+  font-size: 14px;
+  color: #fff;
+  margin: 10px 0;
+}
+
+h3:hover,
+p span:hover {
+  text-decoration: underline;
+}
+
+p{
+  font-family: "Helvetica", "Arial", sans-serif;
+  font-size: 14px;
+  color: #a0a0a0;
+}
+
+.news {
+  max-width: 1300px;
+  margin: 0 auto;
+}
+
+.albums {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  grid-gap: 2em;
+}
+
+.album {
+  display: grid;
+  grid-template-columns: 1fr;
+  max-width: 3500px;
+  grid-gap: 1em;
+}
+
+.album-img {
+  width: 100%;
+  grid-row: 1/2;
+  grid-column: 1/2;
+}
+
+.album ul {
+  width: 100%;
+  height: 100%;
+  grid-row: 1/2;
+  grid-column: 1/2;
+  background-color: black;
+  grid-gap: 1em;
+  padding: 15%;
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr;
+  justify-content: center;
+  align-items: center;
+  
+  opacity: 0;
+}
+
+.album ul:hover {
+  opacity: 0.8;
+}
+
+.album ul li {
+  list-style: none;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.album ul li img {
+  width: 100%;
+  opacity: 0.5;
 }
+
+.play:hover {
+  opacity: 1;
+  transform: scale(1.1);
+  -webkit-transform: scale(1.1);
+  -moz-transform: scale(1.1);
+  -ms-transform: scale(1.1);
+  -o-transform: scale(1.1);
```

Comment:
You don't need vendor prefixes on `transform` - it's supported in all browsers.

Also, the tool which is bundling the css for us actually  uses [autoprefixer](https://www.npmjs.com/package/autoprefixer) so you never need to worry about vendor prefixes when writing css :)